client-app/ui-kit/components/organisms/product-image/vc-product-image.stories.ts

```diff
@@ -0,0 +1,44 @@
+import { VcProductImage } from "..";
+import type { Meta, StoryFn } from "@storybook/vue3";
+
+export default {
+  title: "Components/Organisms/VcProductImage",
+  component: VcProductImage,
+} as Meta<typeof VcProductImage>;
+
+const Template: StoryFn<typeof VcProductImage> = (args) => ({
+  components: { VcProductImage },
+  setup: () => ({ args }),
+  template: '<VcProductImage class="w-32 h-32" v-bind="args" />',
+});
+
+export const Basic = Template.bind({});
+Basic.args = {
+  imgSrc: "https://vcst-dev.paas.govirto.com/cms-content/assets/catalog/VIM70C3/1426013445000_1121979.jpg",
+  alt: "Product",
+};
+
+export const Lazy = Template.bind({});
+Lazy.args = {
+  imgSrc: "https://vcst-dev.paas.govirto.com/cms-content/assets/catalog/VIM70C3/1426013445000_1121979.jpg",
+  alt: "Product",
+};
+
+export const NoImage = Template.bind({});
+NoImage.args = {
+  alt: "Product",
+};
+
+export const Carousel = Template.bind({});
+Carousel.args = {
+  imgSrc: "https://vcst-dev.paas.govirto.com/cms-content/assets/catalog/VIM70C3/1426013445000_1121979.jpg",
```

Comment:
All images should be from this repo. Links from the platform can become invalid.
UI Kit should not know anything about any other storages